pkg/tnf/handlers/owners/owners.go

```diff
@@ -27,6 +27,12 @@ import (
 
 const (
 	owRegex = "(?s)OWNERKIND\n.+"
+	// StatefulSet variable
```

Comment:
using Uppercase in golang, means you are exporting those variables outside of this module. 
it's better to minimize the scope of any variable until it's needed.

use lowercase in all the constants